docs/developers/misc/CollectionsOverview.md

```diff
@@ -0,0 +1,186 @@
+## Collections Overview
+
+### Introduction
+Collections are, as the name suggests, mostly a collection (really a list) of mods. At their core they consist of a `.zip`
+file that contains a `collection.json` file and various other files bundled with it. Most of the mods are stored as a Nexus
+mod ID and fileId, and are therefore sourced from the normal download APIs. 
+
+### Collections Metadata
+Most of the collections metadata is stored and read from the Nexus Mods v2 GraphQL Api. The three main types of ids to be
+aware of are:
+
+- `slug` - a unique identifier for the collection, a 7 character alphanumeric string
```

Comment:
Is this `a-ZA-Z0-9` specifically? Or `a-Z0-9` etc? Might be worth noting.

Weird it's 7 characters, it overflows a 32-bit int, unless it's `a-ZA-Z0-9` and converted to 64-bit int.

Reply:
I think it may actually be `a-z0-9` or at least that's the only examples I've seen

Reply:
Probably is that in this case.
Given they're randomly generated, it's unlikely for them to be anything else.